android/app/src/main/java/com/gpslogger/LocationModule.java

```diff
@@ -63,12 +67,27 @@ public void onStatusChanged(String provider, int status, Bundle extras) {
      * @throws JSONException when the json object cannot be created
      */
     public String createLocationJSON(Location location) throws JSONException {
+
         JSONObject json = new JSONObject();
         json.put("longitude", location.getLongitude());
         json.put("latitude", location.getLatitude());
+	    json.put("altitude", location.getAltitude());
         json.put("timestamp", location.getTime());
         return json.toString();
     }
 
+    /**
+     * returns the current battery level
+     * @return int of the battery level
+     */
+    @ReactMethod
+    public void getBatteryLevel(final Callback battery){
+        IntentFilter ifilter = new IntentFilter(Intent.ACTION_BATTERY_CHANGED);
+        Intent batteryStatus = MainApplication.getAppContext().registerReceiver(null, ifilter);
+        int scale = batteryStatus.getIntExtra(BatteryManager.EXTRA_SCALE, -1);
+        float batlevel = ((float)batteryStatus.getIntExtra(BatteryManager.EXTRA_LEVEL, -1) / (float)scale)* 100.0f;
```

Comment:
I think we should add spaces before and after an operand to improve readability `*`